recipe/meta.yaml

```diff
@@ -151,7 +151,6 @@ requirements:
     - mpfr
     - ntl
     - {{ pin_compatible("numpy") }}
-    - pari
```

Comment:
More of these pins could (should?) be removed since they mostly have run exports.

Should we clean this up @isuruf?